.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
```

Comment:
This one didn't come from the dry-rb rules; any particular reason you've added it here? AFAICR, this isn't a rule I've ever felt stymied by.

Reply:
My mistake, I must have copied from somewhere else. I agree to keep this enabled.